client-docs/README.md

```diff
@@ -76,10 +79,13 @@ type ProjectInfo = {
 
 type ProjectRole = "creator" | "coordinator" | "member";
 
-type Invite = {
-  projectId: ProjectId;
+type PendingInvite = {
+  inviteId: string;
```

Comment:
Currently, this is implemented as a `Buffer`. I can make it a string if that's easier for you.